bin/worker.js

```diff
@@ -119,7 +120,13 @@ co(function *() {
   // Initialize the classes and objects with core functionality used by higher
   // level docker-worker components.
   config.docker = require('../lib/docker')();
-  config.queue = new taskcluster.Queue({ credentials: config.taskcluster });
+  var jsonFromUrl = JSON.parse(fs.readFileSync('test/integration/cancelTaskReference.json'));
```

Comment:
This will be removed once the queue is deployed.  Only for local testing with the queue rewrite.